app/views/service_providers/_form.html.slim

```diff
@@ -1,8 +1,21 @@
+- edit_form ||= false
```

Comment:
let's use `content_for` instead http://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method